src/InstrumentationEngine/ProfilerManager.cpp

```diff
@@ -269,13 +277,11 @@ HRESULT CProfilerManager::GetCorProfilerInfo(
 // return the profiler host instance
 HRESULT CProfilerManager::GetProfilerHost(_Out_ IProfilerManagerHost** ppProfilerManagerHost)
```

Comment:
>GetProfilerHost [](start = 26, length = 15)

Similiar concern here regarding breaking api changes.

Reply:
I think the API became defunct when the profiler manager host extensibility was removed. In all cases, it was made to just return CExtensionsHost, which only implemented IProfilerManagerHost. So calling the Initialize method on it after getting it from this API would either (a) do nothing, or (b) reek havoc on the engine (it probably does the former since it would try to load everything again, but everything was already loaded, so it should no-op).

I got rid of CExtensionsHost because it would be silly to have the engine already know what the instrumentation methods are during attach and pass them (along with their settings) to the CExtensionsHost and have it just pass it back. So I folded CExtensionsHost into CProfilerManager to avoid the middle man.

Any suggestion what should be done for this API?

Reply:
I added a comment in the change log that this method is deprecated and added helpstring to the method in the IDL stating the same.